### PR TITLE
Fix/network ipmi scripts

### PR DESCRIPTION
**Pull request by ColonelPanics**
Fixes made by Steve to get the IPMI config working as well as refactoring so bmc is now included as part of the networks key.

### CODE REVIEW

**Comment by bobwhitelock** [comment]
LGTM if looks good to @ste78!

**Review by bobwhitelock — APPROVED** [review]

File: config/domain.yaml [hunk 1]
```diff
@@ -53,19 +53,23 @@ networks:
...
+    defined: false
+    interface: bmc
+    hostname: "<%= alces.nodename %>.bmc.mgt.<%= domain %>"
+    short_hostname: "<%= alces.nodename %>.bmc"
+    #ip: "10.11.200.<%= alces.index %>"  # XXX uncomment this line when index is working
+    ip: 
+    netmask: 255.255.0.0
+    network: 10.11.0.0
+    gateway: 0.0.0.0
+    bmcpassword: 'Scitjaf4'
```
Comment (sierra-tango-echo) [inline config/domain.yaml]: Can we use Pa55Word as the default bmc password plz

**Review by sierra-tango-echo — DISMISSED** [review]

File: config/domain.yaml (hunk 1 above)
Comment (ColonelPanics) [reply config/domain.yaml]: Have done on my local, will get the change committed when I've got this reinstall sorted!

**Comment by bobwhitelock** [comment]
@ste78 can this be merged now?

**Comment by sierra-tango-echo** [comment]
yep 👍

**Pull request merged**